src/client/client.go

```diff
@@ -85,6 +86,9 @@ type TransactionAPIClient transaction.APIClient
 // DebugClient is an alias of debug.DebugClient
 type DebugClient debug.DebugClient
 
+// LogsClient is an alias of logs.APIClient
```

Comment:
I would not add anything to the external client library. Users should just access our API via the Go gRPC bindings.